src/index.html

```diff
@@ -7,5 +7,37 @@
     <title>Figma Land</title>
   </head>
 
-  <body></body>
+  <body>
+    <!-- START HEADER SECTION -->
+    <header class="header_container">
+      <a href="#" class="header_logo">
```

Comment:
Using javascript:void(0). Check it here https://stackoverflow.com/questions/1291942/what-does-javascriptvoid0-mean. And update the similar cases related

Reply:
Thank you, i updated it